### PR TITLE
API1600  support for Storage Pool

### PR DESCRIPTION
**Pull request by nabhajit-ray**
### Description
[API1600  support for Storage Pool]

### Issues Resolved
[]

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: lib/oneview-sdk/resource/api1600/synergy/storage_pool.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+require_relative '../c7000/storage_pool'
+
+module OneviewSDK
+  module API1600
+    module Synergy
+      # Storage pool resource implementation for API 1600 Synergy
```
Comment (chebroluharika) [inline lib/oneview-sdk/resource/api1600/synergy/storage_pool.rb:17]: please remove space between API and 1600

Comment (nabhajit-ray) [reply lib/oneview-sdk/resource/api1600/synergy/storage_pool.rb]: Done

**Comment by nabhajit-ray** [comment]
> Please add copyright in all spec files that you have added. And update the resourceName from VolumeAttachment to StoragePool in description block in spec files.

Done

File: spec/unit/resource/api800/c7000/storage_pool_spec.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# (C) Copyright 2017 Hewlett Packard Enterprise Development LP
```
Comment (AsisBagga) [inline spec/unit/resource/api800/c7000/storage_pool_spec.rb:1]: please correct this

File: spec/unit/resource/api800/synergy/storage_pool_spec.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# (C) Copyright 2017 Hewlett Packard Enterprise Development LP
```
Comment (AsisBagga) [inline spec/unit/resource/api800/synergy/storage_pool_spec.rb:1]: same

File: spec/unit/resource/api800/c7000/storage_pool_spec.rb (hunk 2 above)
Comment (AsisBagga) [reply spec/unit/resource/api800/c7000/storage_pool_spec.rb]: Checked the files and found them updated

File: spec/unit/resource/api800/synergy/storage_pool_spec.rb (hunk 3 above)
Comment (AsisBagga) [reply spec/unit/resource/api800/synergy/storage_pool_spec.rb]: Checked the files and found them updated

**Pull request merged**